### DISCUSSION TITLE
what the difference is between websocket_message_count and api_message_count

### DISCUSSION THREAD

**Discussion by farshidpg** [beyondcode/laravel-websockets#924] [Q&A]
Hi

what the difference is between websocket_message_count and api_message_count

when broadcast a message to users which of the columns will be updated ?

Also how i can ger peak connection ( max count ) in today and sum of message count like pusher.com dashboard . 
i know laravel eloquent and how edit and show custom data in laravel-websockets dashboard . I need help for work with websocket_message_count and api_message_count sored in db for get max  peak coonection in day and sum of messages that send to user 

tnx

**Comment by farshidpg** [comment]
i get total message count in day with this query , is it true ? 

```
$sum_api_message_count = WebSocketsStatisticsEntry::whereDate('created_at',\Carbon\Carbon::today()->toDateString())->sum('api_message_count');

$sum_websocket_message_count = WebSocketsStatisticsEntry::whereDate('created_at',\Carbon\Carbon::today()->toDateString())->sum('websocket_message_count');

$sum_api_message_count +$sum_websocket_message_count /// total message count
```

**Reply by fredsal** [reply]
try and tell us

**Comment by farshidpg** [comment]
i get total message today ( sum_api_message_count  + sum_websocket_message_count ) but i dontknow is it true ? 
should i sum_api_message_count  + sum_websocket_message_count  or  sum_api_message_count  alone  is my server total message count ?
